extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -392,6 +407,8 @@ export class ConsentInstance {
         return;
       }
 
+      // TODO: enforce metadata limits here (if any)
```

Comment:
What are the metadata limits?

Reply:
We talked about how if we were to add some other metadata fields that were strings (not enums), we should probably enforce a size limit similar to consent string size. Should I remove this comment?

Reply:
Ahhh I see. Thanks for the clarification. That makes sense. Definitely good to keep.